### ISSUE TITLE
Consider looking into the Steam releases of CAVE shmups

### ISSUE THREAD

**Issue by tydog98** [shawngmc/game-extraction-toolbox#15]
Mushihimesama, DoDonPachi Resurrection, and Deathsmiles (the release with the 1st game, I don't know about the bundle with both 1 and 2) on Steam seem to have some sort of ROM files within them (as far as I can tell). Deathsmiles and Mushihimesama use the CAVE CV1000-B arcade system and DoDonPachi Resurrection uses the CAVE CV1000-D system.

**Comment by tydog98** [comment]
I see you've added this to the roadmap, will be closing then

**Comment by shawngmc** [comment]
I'm reopening this until this is implemented (for example, someone else may want to contribute!)

**Comment by shawngmc** [comment]
So, from what I'm seeing, these are likely not ROM-based.

Mushihimesama and DoDonPachi Resurrection each have a sys360.bin file in them, and they reference Xbox. Sure enough, both of these had Xbox 360 releases.

Deathsmiles doesn't have a sys360.bin (but there was a X360 release). However, the .bin files in all 3 titles appear to be some sort of archive format, as most of the files seem to have what looks like archive listings consisting of .TGA files.

@tydog98, is there anything more specific I'm missing? I have my doubts these could be rebuilt to a playable Xbox 360 title, and the value of that at the moment seems rather low.

**Comment by tydog98** [comment]
I do not believe you are missing anything. It seems I misjudged and was a bit excited when I opened this, whoops.
